streaming/src/main/resources/reference.conf

```diff
@@ -1,16 +1,5 @@
 gearpump {
 
-  serializers {
```

Comment:
Why removing this config?

I think GearpumpSerialization is still reading tihs configuration for custom messages.